middle_end/flambda2/types/expand_head.mli

```diff
@@ -91,9 +91,21 @@ val is_bottom : Typing_env.t -> Type_grammar.t -> bool
 
 val is_unknown : Typing_env.t -> Type_grammar.t -> bool
 
+type to_erase =
+  | Everything_not_in of Typing_env.t
+  | All_variables_except of Variable.Set.t
+
+(** This function doesn't descend past an alias type specifying a symbol: the
+    assumption is that such types are already valid in the target environment.
+    (This applies no matter what the setting of [to_erase]).
+
+    If any of the [Name.t]s provided as the "bind-to" names occur already in the
```

Comment:
There's no more argument labelled as "bind-to" so it might be worth not to use this term to avoid causing confusion.

Reply:
Ah, but that was why it was in scare quotes...